tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -337,40 +373,22 @@ export class GraphExecutor implements FunctionExecutor {
   }
 
   disposeIntermediateTensors() {
-    if (!this.intermediateTensors) {
+    if (!this.clonedTensorsMap) {
       return;
     }
-    Object.keys(this.intermediateTensors)
-        .forEach(
-            key => this.intermediateTensors[key].forEach(
-                tensor => tensor.dispose()));
-    this.disposeTensorsMap();
-  }
-
-  private disposeTensorsMap() {
-    if (!this.tensorsMap) {
-      return;
-    }
-    Object.keys(this.tensorsMap).forEach(key => {
-      const tensorArray = this.tensorsMap[key];
-      tensorArray.forEach(tensor => {
-        if (tensor && !tensor.kept && !tensor.isDisposed &&
-            !this.keepIds.has(tensor.id)) {
+    Object.values(this.clonedTensorsMap).forEach(tensorsList => {
+      tensorsList.forEach(tensor => {
+        if (tensor && !tensor.isDisposed) {
```

Comment:
Nit: Google's TS style guide [prefers `for (... of ...)` instead of `.forEach(...)`](https://google.github.io/styleguide/tsguide.html#iterating-containers) (`.map(...)` is still okay when using the resulting value).

Reply:
done